backend/storage/storage_migrations.go

```diff
@@ -57,6 +57,45 @@ type migration struct {
 //
 // In case of even the most minor doubts, consult with the team before adding a new migration, and submit the code to review if needed.
 var migrations = []migration{
+	{Version: "2025-07-15.01", Run: func(_ *Store, conn *sqlite.Conn) error {
```

Comment:
@juligasa ExecScript lets you write multiple statements and execute them in one go (must have the terminating semicolons). Not that it affects the functionality, but it makes it a bit easier to read I think.